pkg/secrets/secrets.go

```diff
@@ -174,6 +179,12 @@ func SecretMounts(filePath, mountLabel, containerWorkingDir string) ([]rspec.Mou
 			if err != nil {
 				return nil, errors.Wrap(err, "error applying correct labels")
 			}
+			if uid != 0 ||
+				gid != 0 {
+				if err := rchown(ctrDirOnHost, uid, gid); err != nil {
+					return nil, err
```

Comment:
should we do a log debug here?

Reply:
I was suggesting a log debug here to show the values from ctrDirOnHost, uid and gid that failed if/when we turned on debug logging.  I think you should still fail with the 'return nil, err' at line 185.